easybuild/easyblocks/o/openmpi.py

```diff
@@ -157,14 +157,24 @@ def load_module(self, *args, **kwargs):
     def sanity_check_step(self):
         """Custom sanity check for OpenMPI."""
 
-        bin_names = ['mpicc', 'mpicxx', 'mpif90', 'mpifort', 'mpirun', 'ompi_info', 'opal_wrapper', 'orterun']
+        bin_names = ['mpicc', 'mpicxx', 'mpif90', 'mpifort', 'ompi_info', 'opal_wrapper']
+        if LooseVersion(self.version) >= LooseVersion('5.0.0'):
+            bin_names.append('prterun')
+        else:
+            bin_names.extend(['mpirun', 'orterun'])
```

Comment:
just curious: what happened to `mpirun` in 5.0.0?

Reply:
Sorry, I made a mistake.
In 4.x, if you would use PRRTE, mpirun would not be available.
However, it reappeared in 5.0.x.